pex/common.py

```diff
@@ -376,25 +386,11 @@ def can_write_dir(path):
     return os.path.isdir(path) and os.access(path, os.R_OK | os.W_OK | os.X_OK)
 
 
-def touch(file, times=None):
```

Comment:
We never used the `times` kwarg. It's really tricky to get right with type hints, so I took the liberty to simplify this function.

I figure that we can add back the `times` part if we need it.